index.html

```diff
@@ -146,10 +146,10 @@ <h1 class="short-text sidebar-header">
 
   <script type="text/javascript" src="data/africa_data.js" charset="utf-8"></script>
   <script type="text/javascript" src="data/cases_points.js" charset="utf-8"></script>
-  <script type="text/javascript" src="js/legend_colors_fns.js?v=0.0.1"></script>
+  <script type="text/javascript" src="js/legend_colors_fns.js?v=0.0.2"></script>
   <script type="text/javascript" src="js/legends.js?v=2"></script>
-  <script type="text/javascript" src="js/layers_object.js?v=0.0.1"></script>
-  <script type="text/javascript" src="js/layers_fn.js?v=2.1.4"></script>
+  <script type="text/javascript" src="js/layers_object.js?v=0.0.2"></script>
+  <script type="text/javascript" src="js/layers_fn.js?v=2.1.5"></script>
```

Comment:
I'm about to merge this so don't push anything to the kerwa_point branch, but was there something you'd added to the layers_fn.js file, @lynnnakiganda ?

Reply:
Am done with this branch so I won't be pushing anything else. I was trying to check what I added to that file but I can't find anything  so that version change was a mistake

Reply:
Am done with this branch so I won't be pushing anything else. I was trying to check what I added to that file but I can't find anything  so that version change was a mistake